gps/index.html

```diff
@@ -10,27 +10,27 @@
 date: 2018-04-09
 categories: []
 ---
-<h2 class="text-left"><i class="fas fa-fw fa-question"></i> What is a Reference Network</h2>
+<h2 class="text-left"><i class="fas fa-fw fa-question"></i> What Is a Reference Network?</h2>
 <img class="inline-text-right outline" src="images/turn.png" alt="TURN GPS Network">
-<p>A GPS network is a high precision Global Navigation Satellite System (GNSS) providing real time corrections and data for post processing. A GNSS network consists of permanently located GPS receivers, installed across an area, that generate real time high accuracy GPS positioning. Basically, all of the permanently installed GPS receivers send their data to a central system via the internet, that analyzes the input and adjusts and corrects the data to provide the most accurate locations usually down to centimeter level. Our networks use the Trimble PIVOT VRS Platform. The generated data is sent through a wireless connection using the Networked Transport of RTCM via Internet Protocol (NTRIP).</p>
+<p>A GPS network is a high-precision Global Navigation Satellite System (GNSS) providing real-time corrections and data for post processing. A GNSS network consists of permanently located GPS receivers, installed across an area, that generate real-time, high-accuracy GPS positioning. Basically, all of the permanently installed GPS receivers send their data to a central system via the internet. This central system analyzes the input and adjusts and corrects the data to provide the most accurate locations, usually down to centimeter level. Our networks use the Trimble PIVOT VRS Platform. The generated data is sent through a wireless connection using the Networked Transport of RTCM via Internet Protocol (NTRIP).</p>
 
 
-<h2 class="text-left"><i class="fas fa-fw fa-user-plus"></i> Who should be using a Network</h2>
-<p>Any person or company that wants or requires precision GPS information should be using a GPS Network. The most common applications including surveying, engineering, construction, GPS controlled machinery and GIS data collection.</p>
+<h2 class="text-left"><i class="fas fa-fw fa-user-plus"></i> Who Should Be Using a Network?</h2>
+<p>Any person or company that wants or requires precision GPS information should be using a GPS network. The most common applications of a GPS network include surveying, engineering, construction, GPS-controlled machinery, and GIS data collection.</p>
```

Comment:
Did I understand the "of a GPS network" correctly?

Reply:
yup